### ISSUE TITLE
Enhance certain button (Update/Edit/Delete/Cancel/etc.) placements for better UX

### ISSUE THREAD

**Issue by joewesch** [nautobot/nautobot#5989]
### As ...

Nelly - Network Engineer

### I want ...

To be able to see and quickly click any of the action buttons (Update/Edit/Delete/Cancel/etc.) on a form or table regardless of how long the form is or how many items are shown in the table.

### So that ...

I don't need to scroll down to the bottom of the page to find the button.

### I know this is done when...

- I can edit a field near the top of the page on a Device and click Update without having to scroll
- I can have a long table (e.g. 1000 items) and select 1+ item(s) and click Edit Selected or Delete Selected buttons without having to scroll
- I can filter my items and do a bulk delete and not have to scroll to the bottom of the page to click `Delete these # items` button.

### Optional - Feature groups this request pertains to.

- [ ] Automation
- [ ] Circuits
- [ ] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [ ] Apps (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [X] UI/UX
- [ ] Documentation
- [ ] Other (not directly a platform feature)

### Database Changes

None

### External Dependencies

Unknown, but probably not since it's UI based.

**Comment by HanlinMiao** [comment]
Something like a sticky button that sticks to the top/bottom of the browser viewport comes to mind

**Comment by gsnider2195** [comment]
Possible duplicate of #5318?

**Comment by joewesch** [comment]
> Possible duplicate of #5318?

Indeed. Thanks, I didn't find that one in my original search I guess.
